stdlib/Profile/src/Profile.jl

```diff
@@ -115,13 +121,18 @@ The keyword arguments can be any combination of:
  - `maxdepth` -- Limits the depth higher than `maxdepth` in the `:tree` format.
 
  - `sortedby` -- Controls the order in `:flat` format. `:filefuncline` (default) sorts by the source
-    line, whereas `:count` sorts in order of number of collected samples.
+    line, `:count` sorts in order of number of collected samples, and `:overhead` sorts by the number of samples
+    incurred by each function by itself.
 
  - `noisefloor` -- Limits frames that exceed the heuristic noise floor of the sample (only applies to format `:tree`).
     A suggested value to try for this is 2.0 (the default is 0). This parameter hides samples for which `n <= noisefloor * √N`,
     where `n` is the number of samples on this line, and `N` is the number of samples for the callee.
 
  - `mincount` -- Limits the printout to only those lines with at least `mincount` occurrences.
+
+ - `recur` -- Controls the recursion handling in `:tree` format. `off` (default) prints the tree as normal. `flat` instead
+    compresses any recursion (by ip), showing the approximate effect of converting any self-recursion into an iterator.
```

Comment:
Saying it's "like an iterator" leaves ambiguous the case where `f` calls `g` calls `h` calls `f`... and one wonders where this chain gets broken.

Reply:
I think that doesn't really matter (where it gets broken), since the tree should look identical in that case